tests/utils.ts

```diff
@@ -0,0 +1,42 @@
+import * as request from 'supertest';
+import { getConnection } from 'typeorm';
+import * as winston from 'winston';
+
+import createServer from '../src/server';
+import { createConnection } from '../src/db';
+
+export const setup = createConnection;
+export const clearDb = () => getConnection().synchronize(true);
```

Comment:
nit: don't split export statements.

Reply:
What do you mean? you prefer `export default { setup, clearDb, get, post }` over prefixing `export` keyword?

Reply:
no no, I mean keep the exports together, like in continuous lines.